components/script/dom/subtlecrypto.rs

```diff
@@ -3202,6 +3202,7 @@ impl JsonWebKeyExt for JsonWebKey {
         // (It is given as a method paramter.)
 
         // Step 2. Let json be the Unicode string that results from interpreting data according to UTF-8.
+        #[allow(clippy::incompatible_msrv)]
```

Comment:
Ah, this will break when we try to merge on CI, since we check that out MSRV is accurate. Let's use https://doc.rust-lang.org/stable/std/string/struct.String.html#method.from_utf8_lossy instead.